src/interfaces/error-i.ts

```diff
@@ -8,3 +8,10 @@ export interface ErrorI {
   statusCode: number | null,
   type?: ErrorCodeE
 }
+
+export const AccountAccessError: ErrorI = {
```

Comment:
shall we move this const outside of the interface definition? like to the account store.

Reply:
I would keep this here unless you have specific reason to move it out ?